stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Nicolas Dorier and BTCPayServer – self hosted Bitcoin and Lightning payments 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+media: https://stephanlivera.com/download-episode/955/48.mp3
+---
+podcast: https://stephanlivera.com/episode/48/
+## Introdcution
```

Comment:
These headers need space around them. This is not valid markdown.

Reply:
Thanks. I updated according to your suggestion

Reply:
why do we need these headers at all? They don't appear to add anything to the transcript